apps/kur_kards_web/assets/css/app.scss

```diff
@@ -29,3 +29,15 @@
 .alert:empty {
   display: none;
 }
+
+#message-list {
```

Comment:
Thinking of using emotionjs instead of scss but not decided currently and this file might be removed in the future anyway

Reply:
I am part of the Tailwindcss religious cult so your way of doing is already heretical ahaha